legion/tests/legion_test_utils.py

```diff
@@ -83,6 +96,118 @@ def patch_environ(values, flush_existence=False):
     return patch('os.environ', new_values)
 
 
+@contextlib.contextmanager
+def persist_swagger_function_response_to_file(function, response_codes):
```

Comment:
@kirillmakhonin do we really need this?

Reply:
sure, I'll simplify this function and remove possibility to pass response_codes as a list. Also argument name can be changed, but I'm not sure which will be correct.